### DISCUSSION TITLE
try catch question

### DISCUSSION THREAD

**Discussion by xiaoxiji1234** [lowdefy/lowdefy#1163] [Questions and Help]
Hi, I used `try catch` in the `event` to process the error message of `Request`, but I couldn't catch  the `error message` in `catch`. Is there a specific example for reference?

**Comment by Gervwyk** [comment]
Hi @xiaoxiji1234 
If your request does not return a `5xx` or `4xx` error code, the request action will not throw. Your need to throw in the action chain to stop the current action chain and evoke the `catch` actions. You can also use the [`Throw`](https://docs.lowdefy.com/Throw) action to throw an error in an action chain based on config conditions.

**Reply by xiaoxiji1234** [reply]
Hi, @Gervwyk 
My request has an error code. I want to capture this error code as a judgment condition. For example, I get error code 401. How can I realize it?
error:
![image](https://user-images.githubusercontent.com/72787010/168219850-f4bf30c6-25f5-41ed-9c42-640981f259db.png)
